src/main/java/com/conveyal/r5/analyst/cluster/RegionalWorkResult.java

```diff
@@ -65,7 +65,12 @@ public RegionalWorkResult(OneOriginResult result, RegionalTask task) {
         this.travelTimeValues = result.travelTimes == null ? null : result.travelTimes.values;
         this.accessibilityValues = result.accessibility == null ? null : result.accessibility.getIntValues();
         this.pathResult = result.paths == null ? null : result.paths.summarizeIterations(PathResult.Stat.MINIMUM);
-        this.opportunitiesPerMinute = result.density == null ? null : result.density.opportunitiesPerMinute;
+        if (result.density != null) {
+            this.opportunitiesPerMinute = result.density.opportunitiesPerMinute;
+            if (task.originPointSet == null) {
+                this.accessibilityValues = result.density.calculateDualAccessGrid();
```

Comment:
Relatedly, it looks like this could try to generate dual access grids even when no dual thresholds are specified, resulting in a null pointer exception in `calculateDualAccessGrid()`. However, here I think the `originPointSet == null` condition is excluding the case where we can calculate density without dual access (freeform pointsets specified as origins, producing CSV output).

Reply:
While this code path should not be hit without `dualAccessThresholds` being set, I added an additional check in the PR here: https://github.com/conveyal/r5/pull/961